src/index.tsx

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+import { AuthProvider } from './context/AuthContext';
 
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <App />
+  <AuthProvider>
```

Comment:
Providery lepiej jak sa w App. Index najlepiej jak pozostaje nieskalany. 
Najlepiej to zrobimy sobie na lekcji nowy component ContextProvider, zeby bylo ladniej ;)